components/rhs_thread/rhs_thread.jsx

```diff
@@ -89,6 +88,9 @@ export default class RhsThread extends React.Component {
     componentDidMount() {
         this.scrollToBottom();
         window.addEventListener('resize', this.handleResize);
+        if (this.props.posts.length < (Utils.getRootPost(this.props.posts).reply_count + 1)) {
+            this.props.actions.getPostThread(this.props.selected.id, true);
```

Comment:
Still seems hacky wrt UX. How was this supposed to be handled with improvements with threads?. 

This happens quite often and showing some posts we have in local state with broken context i.e missing posts in between first followed by the API calls seems odd.

What was expected in UI when an incomplete thread is loading?

Reply:
@Willyfrog @reflog above question is not a blocking one.

Reply:
Ya. This is going to cause jumping when the RHS opens in some situations. We probably need to handle the loading condition.